dev/packages/alpha/system/0.0.1/dataset/entropy/fields/fields.yml

```diff
@@ -0,0 +1,13 @@
+- name: system.entropy
+  type: group
+  release: ga
+  fields:
+  - name: available_bits
+    type: long
+    description: |
+      The available bits of entropy
+  - name: pct
+    type: scaled_float
+    format: percent
```

Comment:
@skh @neptunian Do we already support formatters in our kibana index patterns?
@fearful-symmetry Nothing to change here.